zookeeper-contrib/zookeeper-contrib-loggraph/src/main/java/org/apache/zookeeper/graph/Log4JSource.java

```diff
@@ -293,7 +293,7 @@ private void init() throws IOException {
 	if ((line != null) && (m = timep.matcher(line)).lookingAt()) {
 	    starttime = timestampFromText(dateformat, m.group(1));
 	} else {
-	    throw new IOException("Invalid log4j format. First line doesn't start with time");
+	    throw new IOException("Invalid log format. First line doesn't start with time");
```

Comment:
This class has the name of "Log4JSource.java". I think we should rename it to "LogSource".

I am not sure what is the purpose of this class, but this seems to parse some log messages. Am I right, that you defined 'log4j1-like' log format in the logback.xml file below?

Reply:
Yeah, I used the logback translator tool to convert the config file, so the format should be the same.
Renaming the file also makes sense, I'll look into that. Thanks.